src/parser/jobs/smn/modules/Ruin2.js

```diff
@@ -94,7 +95,7 @@ export default class Ruin2 extends Module {
 			this._all.push(event)
 			// Explicitly setting the ogcd tracker to true while bahamut is out,
 			// we don't want to fault people for using R2 for WWs during bahamut.
-			this._ogcdUsed = this.gauge.bahamutSummoned()
+			this._ogcdUsed = (this.pets.getCurrentPet() === PETS.DEMI_BAHAMUT.id)
```

Comment:
Probably won't have a huge effect given popular bahamut rotations, but this _probably_ isn't worth being as lenient about any more.

Reply:
My understanding is that a few instant Ruins are used at the end of Bahamut to ensure all 8 WW, even though you don't have anything to weave.  Depending on how many Further Ruin stacks you have, these may be Ruin 2 or Ruin 4.